accounts/abi/bind/backends/simulated.go

```diff
@@ -24,7 +24,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/spruce-solutions/go-quai"
+	ethereum "github.com/spruce-solutions/go-quai"
```

Comment:
Why is this adding Ethereum?

Reply:
it has been adding ethereum on all of our PRs now

https://github.com/spruce-solutions/go-quai/pull/139/files check the api_backend.go. I will see why that's happening